tests/attested_tls/oe_enclave/enc/server.cpp

```diff
@@ -0,0 +1,829 @@
+// Copyright Microsoft. 
```

Comment:
Can you rename the file to "tlssrv.cpp"?

Reply:
sure, I will change it.